.github/workflows/staging-push.yml

```diff
@@ -1,9 +1,9 @@
 name: Testops Report PUSH 
 
 on:
-  push:
-    branches:
-      - master 
+  # push:
```

Comment:
Commenting for now, so that it does not run on pushes. Eventually this github action will be removed-renamed